cirq-core/cirq/protocols/json_test_data/CircuitOperation.json

```diff
@@ -292,7 +292,8 @@
           ]
         },
         "parent_path": [],
-        "repetition_ids": null
+        "repetition_ids": null,
+        "use_repetition_ids": false
```

Comment:
We should probably include a case where this is true (and thus absent) as well, just to ensure backwards compatibility behavior works properly.

Reply:
I added such a test case to the repr file. Since the json field is only emitted when it's false, there are no changes to the json file here. Wanted to make sure that's what you expected. (The other option would be to have the default value in code be `CircuitOperation.use_repetition_ids = None` and emit the json iff it's explicitly set to either `True` or `False`. But I don't *think* that's what you were suggesting).

Reply:
Yup, that was the expectation.